frontend/src/Pages/PostDetails.module.css

```diff
@@ -0,0 +1,31 @@
+:root {
+  --bg-main: #606c38;
+  --bg-two: #283618;
+  --bg-three: #d3bfa9;
+  --bg-four: #dda15e;
+  --color-main: #f8f8ff;
+  --color-two: #f8f8ff;
+  --color-three: #444444;
+  --color-four: #444444;
+  --page-break: 820px;
+  --padding-mobile: 2%;
+  --padding-desktop-leftright: 4%;
+  --padding-desktop-above: 0.1%;
+}
+
+h1 {
```

Comment:
Funktioniert das? Sollte eigentlich nicht funktionieren. Eigentlich brauchst du bei CSS Modules immer einen ClassName, den du definierst, also in deinem Fall von mir aus ".title", den du dann natürlich auch in der "PostDetail.jsx" dem entsprechenden Element anhängst.